poi.config.js

```diff
@@ -37,6 +38,7 @@ const devServer = {
   },
   proxy: {
     '/api': { target: serverUrl },
+    ...(STORAGE_PROXY === 'local' ? { '/proxy': serverUrl } : {}),
```

Comment:
Should the base URL for the proxy (local or cf) always be exposed in order to be able to construct asset url from client based on key?